File: Level-1/68.java
class Solution {
    public int solution(String t, String p) {
        int answer = 0;
        
        int tlen = t.length();
        int plen = p.length();
        
        for(int i = 0; i <= tlen - plen; i++) {
            Long num = Long.parseLong(t.substring(i, i + plen));
            if(num <= Long.parseLong(p)) {
                answer++;
            }
        }
        
        return answer;
    }
}

File: Level-1/68.js
function solution(t, p) {
    const length = p.length;
    let answer = 0;
    
    for(let i = 0; i <= t.length - length; i++) {
        const num = Number(t.substring(i, i + length));
        if(num <= Number(p)) answer++;
    }
    
    return answer;
}
